src/app/dataset-block/metadata-block/components/event-details/components/add-data-event/add-data-event.component.html

```diff
@@ -0,0 +1,144 @@
+<div class="d-flex flex-column gap-4">
+    <div
+        *ngIf="event?.inputCheckpoint"
```

Comment:
`event` cannot be null, looking on it's type. Why do you need those ? checks

Reply:
Removed ckecks "?"